src/prefect/tasks/gcp/bigquery.py

```diff
@@ -385,10 +385,21 @@ def run(
                 table_ref,
                 location=location,
                 job_config=job_config,
-            ).result()  # block until job is finished
+            )
+
+            # Start the job and wait for it to complete and get the result
+            load_job.result()
         except Exception as exception:
-            for error in load_job.errors:
-                self.logger(error)
+            self.logger.error(exception)
+
+            if exception.errors is not None:
```

Comment:
I think we can remove this. I'm not sure what exception gets thrown here (and if it has an `errors` attribute, but we're already logging the full exception.